library/Zend/Test/DbStatement.php

```diff
@@ -140,7 +140,11 @@ public function setRowCount($rowCount)
      */
     public function append($row)
     {
-        $this->_columnCount = count($row);
+        if (!is_array($row)) {
```

Comment:
`$row`, in the tests at least, can be just a string, which `count` returns `1` for. This should act the same, just avoid the warning thrown by php 7.2+.